### PR TITLE
Todo list

### PR DESCRIPTION
**Pull request by Ntwali-Josue**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "webpack-demo",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```
Comment (dasileker) [inline package.json:9]: - [ ] The script should also having the **`start`** script like you did in the lesson before you start

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="main.js"></script>
+  <link rel="stylesheet" href="style.css">
+  <title>Todo List</title>
+</head>
+<body>
+      <ul class="item"></ul>
+  <script src="./index.js"></script>
```
Comment (dasileker) [inline src/index.html:13]: - [ ] You should at least have an input field this is the Original 👇
input

- And this is your project 👇 you should follow the Microverse layout.
you

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Ntwali-Josue **, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Follow the structure  that Microverse has 
total


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (Ntwali-Josue) [reply package.json:10]: Thanks for the feedback. I have worked on it.

File: src/index.html (hunk 2 above)
Comment (Ntwali-Josue) [reply src/index.html]: Thanks for this feedback. I have worked on it as you see here: 
image

Comment (rloterh) [reply src/index.html]: This issue is well resolved! ✔️

File: package.json (hunk 1 above)
Comment (rloterh) [reply package.json]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Karangwa, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 :shipit: .
Congratulations! 🎉 🎉 🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ntwali-Josue** [comment]
Thank you for approving my project @rloterh 💯

**Pull request merged**